test/check-storage-basic

```diff
@@ -304,53 +315,95 @@ class TestStorage(anacondalib.VirtInstallMachineCase, StorageHelpers):
-@nondestructive
-class TestStorageExtraDisks(anacondalib.VirtInstallMachineCase, StorageHelpers):
-    efi = False
+class TestStorageGuided(VirtInstallMachineCase, StorageHelpers):
+    @test_plan("https://fedoraproject.org/wiki/QA:Testcase_partitioning_guided_encrypted")
+    def testGuidedEncrypted(self):
+        b = self.browser
+        i = Installer(b, self.machine)
+        s = Storage(b, self.machine)
+        p = Password(b, s.encryption_id_prefix)
+        pro = Progress(b)
+
+        i.open()
+        i.reach(i.steps.INSTALLATION_METHOD)
+
+        # Check the default mode
+        s.check_partitioning_selected("erase-all")
+
+        i.next()
+        # Disk Encryption
+        s.set_encryption_selected(True)
+        strong_password = "Rwce82ybF7dXtCzFumanchu!!!!!!!!"
+        p.set_password(strong_password)
+        p.set_password_confirm(strong_password)
+
+        # Reboot and check installed system
+        i.reach(i.steps.REVIEW)
+        i.begin_installation()
+        pro.wait_done()
+        s.unlock_storage_on_boot(strong_password)
+        self.handleReboot()
+        s.check_post_install_encryption_enabled()
 
-    def testLocalDisksSyncNew(self):
+    @test_plan("https://fedoraproject.org/wiki/QA:Testcase_partitioning_guided_multi_select")
+    def testGuidedMultiSelect(self):
         b = self.browser
         m = self.machine
-        i = Installer(b, m)
-        s = Storage(b, m)
+        i = Installer(b, self.machine)
+        s = Storage(b, self.machine)
+        p = Progress(b)
+
+        self.add_disk(15, os.path.join(BOTS_DIR, "./images/debian-stable"))
 
         i.open()
         i.reach(i.steps.INSTALLATION_METHOD)
+        s.rescan_disks()
 
-        s.wait_no_disks_detected_not_present()
+        s.check_disk_selected("vda", True)
+        s.check_disk_selected("vdb", False)
 
-        s.check_single_disk_destination("vda")
+        i.reach(i.steps.REVIEW)
+        i.begin_installation()
+        p.wait_done()
 
-        # This attaches a disk to the running VM
-        # However, since the storage module initialization is long completed
-        # the newly added disk, will not be visible in the UI,
-        # until the test clicks on the re-scan button
-        dev = self.add_ram_disk(2)
-        dev = dev.split("/")[-1]
+        self.handleReboot()
 
-        s.rescan_disks()
+        # Expect the new OS is the default grub entry
+        pretty_name = get_pretty_name(m)
+        self.assertIn("Fedora Linux", pretty_name)
 
-        s.check_disk_visible("vda")
-        s.check_disk_visible(dev)
+        m.reboot()
+        self.selectBootMenuEntry(2)
+        # FIXME: Debian will not boot unless we remove the vda disk
+        # pretty_name = get_pretty_name(m)
+        # self.assertIn("Debian GNU/Linux", pretty_name)
 
-        s.wait_no_disks_detected_not_present()
+    @test_plan("https://fedoraproject.org/wiki/QA:Testcase_partitioning_guided_multi_empty_all")
```

Comment:
I don't see this Expected result tested here:
"The created partition layout should span all the target devices."

Reply:
I wonder if it actually belongs under this expected result:
"The expected partition layout should be created on the target device(s)."
which is not checked as well, but maybe on purpose.